ui/js/ui.history.js

```diff
@@ -64,6 +59,27 @@ var UI = (function(UI, $, undefined) {
 
           modal = $modal.remodal(options);
           modal.open();
+        }
+
+        iota.api.isReattachable(inputAddresses, function (error, isReattachable) {
+          if (error) {
+            return
+          }
+          if (inputAddresses.length === 0) {
+            return iota.api.findTransactionObjects({bundles: [transactions[0].bundle]}, function (err, txs) {
+              if (err) {
+                return
+              }
+              iota.api.getLatestInclusion(txs.filter(tx => tx.currentIndex === 0).map(tx => tx.hash), function (err, states) {
+                if (err) {
+                  return
+                }
+                isReattachable = states.indexOf(true) === -1
+                renderBundleModal(isReattachable)
+              })
+            })
+          }
+          renderBundleModal(isReattachable)
```

Comment:
this is missing an else I believe.

Reply:
it returns above inside the if block (L69). I actually think of this often, what's the best coding style...

Reply:
yes. but if `isReattachable` is true already then you don't need to call `findTransactions`!?

Reply:
nevermind. too late here, you're right.